liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/DropdownMenuDTO.kt

```diff
@@ -0,0 +1,290 @@
+package org.phoenixframework.liveview.data.dto
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.unit.DpOffset
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.PopupProperties
+import androidx.compose.ui.window.SecureFlagPolicy
+import org.phoenixframework.liveview.data.constants.Attrs.attrClippingEnabled
+import org.phoenixframework.liveview.data.constants.Attrs.attrDismissOnBackPress
+import org.phoenixframework.liveview.data.constants.Attrs.attrDismissOnClickOutside
+import org.phoenixframework.liveview.data.constants.Attrs.attrExcludeFromSystemGesture
+import org.phoenixframework.liveview.data.constants.Attrs.attrExpanded
+import org.phoenixframework.liveview.data.constants.Attrs.attrFocusable
+import org.phoenixframework.liveview.data.constants.Attrs.attrOffset
+import org.phoenixframework.liveview.data.constants.Attrs.attrPhxClick
+import org.phoenixframework.liveview.data.constants.Attrs.attrSecurePolicy
+import org.phoenixframework.liveview.data.constants.Attrs.attrUsePlatformDefaultWidth
+import org.phoenixframework.liveview.data.core.CoreAttribute
+import org.phoenixframework.liveview.domain.base.ComposableBuilder
+import org.phoenixframework.liveview.domain.base.ComposableView
+import org.phoenixframework.liveview.domain.base.ComposableViewFactory
+import org.phoenixframework.liveview.domain.base.PushEvent
+import org.phoenixframework.liveview.domain.factory.ComposableTreeNode
+import org.phoenixframework.liveview.ui.phx_components.PhxLiveView
+
+/**
+ * Material Design dropdown menu. Menus display a list of choices on a temporary surface.
+ * They appear when users interact with a button, action, or other control.
+ * You must use the `expanded` property to show/hide the `DropdownMenu` and define the dismiss
+ * event to hide it:
+ * ```
+ * def handle_event("showPopup", _params, socket) do
+ *   {:noreply, assign(socket, :showPopup, true)}
+ * end
+ *
+ * def handle_event("hidePopup", _params, socket) do
+ *   {:noreply, assign(socket, :showPopup, false)}
+ * end
+ *
+ * def handle_event("onMenuOptionClick", _params, socket) do
+ *   {:noreply, socket |> assign(:menuOption, _params) |> assign(:showPopup, false) }
```

Comment:
nit: I think an underscore typically indicates the value will not be used in the function body.